public/controllers/agent/agents-preview.js

```diff
@@ -92,6 +94,14 @@ export class AgentsPreviewController {
         !parameters.items.length > 0 && !parameters.filters.length;
     });
 
+    this.$scope.offsetTimestamp = (text, time) => {
+      try {
+        return text + this.timeService.offset(time);
+      } catch (error) {
+        return '';
```

Comment:
What the view looks when returning `''`? What happens?

Reply:
Nothing was shown. Now I have changed it so that it returns the date and the base timezone. `return '${time} (UTC)';`